### DISCUSSION TITLE
Famland资金流向图

### DISCUSSION THREAD

**Discussion by otto-001** [famland/website#48] [Show and tell]
![image](https://user-images.githubusercontent.com/97232831/165560664-2d0dee15-ce27-464c-9d46-558006a04b0e.png)

**Comment by yancanlou** [comment]
和田玉的开采与合田部分的分流情况没补上

**Reply by cryptosoul-vioiv** [reply]
> 和田玉的开采与合田部分的分流情况没补上

是的，缺少一点内容
